docker/from-packages/scripts/start-collabora-online.pl

```diff
@@ -34,9 +35,42 @@ ()
 
                 $output .= "                </group>\n";
             }
+        } elsif (/^domain/) {
+            my $value = $ENV{$_};
+            $message .= "WARNING: The 'domain=$value' is deprecated.\n For your convenience, we interpret it as the following:\n";
+            my @hosts = split('\|', $value);
+            if (@hosts) {
+                my $i = 0;
+                foreach (@hosts) {
+                    $i++;
```

Comment:
Should we also add a check here for the case there is some form of regexp in the domain= ?  Or will it work in the alias_groups?

Reply:
it will work